app/views/admin/tags/index.html.erb

```diff
@@ -1,27 +1,29 @@
 <div class="flex items-center mb-6">
-  <ul class="nav nav-pills">
-    <li class="nav-item">
-      <a href="/admin/tags" class="nav-link <%= "active" if params[:state].blank? %>">All</a>
-    </li>
-    <li class="nav-item">
-      <a href="/admin/tags?state=supported" class="nav-link <%= "active" if params[:state] == "supported" %>">Supported</a>
-    </li>
-    <li class="nav-item">
-      <a href="/admin/tags?state=unsupported" class="nav-link <%= "active" if params[:state] == "unsupported" %>">Unsupported</a>
-    </li>
-  </ul>
+  <div class="crayons-tabs">
```

Comment:
Thank you for taking the time to make the switch to Crayons here and throughout your changes -- it looks great!